clients/venice-admin-tool/src/main/java/com/linkedin/venice/Command.java

```diff
@@ -474,6 +363,11 @@ public enum Command {
       "dump-ingestion-state",
       "Dump the real-time ingestion state for a certain store version in a certain storage node",
       new Arg[] { SERVER_URL, STORE, VERSION }, new Arg[] { PARTITION }
+  ),
+  ADD_CHANGE_CAPTURE_VIEW(
+      "add-change-capture-view",
+      "Add change capture to a given store. This will enable all required configs including AA replication and chunking. View name argument is optional, default view name is changeCaptureView",
+      new Arg[] { URL, STORE, CLUSTER }, new Arg[] { VIEW_NAME }
```

Comment:
What is the name for, BTW...?

Reply:
So the way views are configured for a store are a map of Map<String, ViewConfig>.  The intention of the key is to be able to have a handle for which view you want to edit.  This way, it's conceivable that a store could have multiple views of the same type, but perhaps with different configurations (admittedly for the change capture view, this wouldn't work, or at least, wouldn't work in a useful way).  The optional argument is if you wanted to override the default name for some reason.

....Though... maybe this doesn't make sense now that I type it out.  Maybe it'd be better to just not have it.